pkg/certificate/controller.go

```diff
@@ -101,21 +102,36 @@ func (m *Manager) Reconcile(request reconcile.Request) (reconcile.Result, error)
 		}
```

Comment:
Regarding Reconsile, what happens if we delete one of the services?
For example, let's say there are 2 services, that were created when the server was created, with 1year expiration. After 0.5year,  one of the services are deleted, and recreated by the controller. 
1. Will we notice it in this reconcile?
2. if  we do, do we recreate all of the services at once?

Reply:
I don't know if you can remove a Services that is references by a webhook configuration, maybe kube-apiserver prevent it, anyhow Service will be re-created and secret still valid, I suppose the services will be reacreated by CNAO right ? we don't re-create them here.

Reply:
you're right, it's not our job to recreate them here. but the'll get the cert with the existing expiration. so no issue here, just wanted to make sure I understand